src/Adapter/Country/CommandHandler/AddCountryHandler.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
Same problems with header found above.

You may be interested by this command : `vendor/bin/header-stamp prestashop:licenses:update`